### PR TITLE
feat(reg/tickets): update content to 2024 in ticket page

### PR DESCRIPTION
**Pull request by SivanYeh**
## Types of changes

* **New feature**

## Description

Release 2024 Ticket Info 

## Steps to Test This Pull Request

1. Go to “http://localhost:3000/2024/zh-hant/registration/tickets"
2. Check if content has been updated and matched the document: https://docs.google.com/document/d/1fqyk_UEdX1k44xjHzUGevWDQ8PgsIQ9_T32DmTQ_bRw/edit.
3. Repeat step 1-2 for english version “http://localhost:3000/2024/en-us/registration/tickets”

## Related Issue

#523

## Screenshot
image

image

### CODE REVIEW

File: configs/pageTickets.js
Comment (rockleona) [inline configs/pageTickets.js:1]: 這邊的 KKTIX 連結 (value) 都沒有更新到ㄟ

File: configs/pageTickets.js [hunk 1]
```diff
@@ -102,44 +94,31 @@ export const ticketInfoConfig = [
...
+        statusI18nKey: ticketSellingStatus.COMING_SOON,
+        link: pageLinks.kktixCorporate2024,
     },
     {
         titleI18nKey: 'tickets.groupBuyCorporate',
-        priceI18nKey: 'terms.freeToApply',
+        prices: [
+            {
+                titleI18nKey: 'sellingStage.regularStage',
+                price: 'NT$ 5,222',
```
Comment (rockleona) [inline configs/pageTickets.js:109]:
```suggestion
                price: 'NT$ 5,220',
```

**Review by rockleona — CHANGES_REQUESTED** [review]
Review 1st Round!

**Comment by SivanYeh** [comment]
@rockleona fixed! 感謝挑錯!!

**Review by rockleona — APPROVED** [review]
LGTM! 可能要提醒下一個階段處理這個頁面的志工留意一下有關 KKTIX 的連結

**Pull request merged**